Not supported on OpenBSD, but guacd seems to run OK without this anyway.

Index: src/libguac/socket-wsa.c
--- src/libguac/socket-wsa.c.orig
+++ src/libguac/socket-wsa.c
@@ -427,7 +427,9 @@ guac_socket* guac_socket_open_wsa(SOCKET sock) {
     socket->data = data;
 
     pthread_mutexattr_init(&lock_attributes);
+#ifndef __OpenBSD__
     pthread_mutexattr_setpshared(&lock_attributes, PTHREAD_PROCESS_SHARED);
+#endif
 
     /* Init locks */
     pthread_mutex_init(&(data->socket_lock), &lock_attributes);
